projects/go-lib/src/lib/components/go-radio/go-radio-button.component.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../../styles/variables';
+
+input[type='radio'] {
```

Comment:
Instead of using the element selector in these, a BEM class should be used. Something like `.go-radio__input` would suffice.

Reply:
updated